volume/drivers/aws/aws.go

```diff
@@ -113,11 +113,17 @@ func (d *Driver) freeDevices() (string, string, error) {
 		return "", "", err
 	}
 	devPrefix := "/dev/sd"
+
 	for _, dev := range self.BlockDeviceMappings {
 		if dev.DeviceName == nil {
 			return "", "", fmt.Errorf("Nil device name")
 		}
 		devName := *dev.DeviceName
+
+		// sda1 is reserved for the root device, skip
```

Comment:
Per AWS docs EC instances have the root mounted at /dev/sda1. This label should be skipped.